lib/oauthenticator/version.rb

```diff
@@ -1,5 +1,5 @@
 # OAuthenticator 
 module OAuthenticator
   # OAuthenticator::VERSION
-  VERSION = "1.3.1"
+  VERSION = "1.3.4"
```

Comment:
I will merge the previous commit. it is preferable to me to manage versions outside of pull requests.